pkg/ocp/machinesets.go

```diff
@@ -69,15 +69,14 @@ func (msd *k8sMachineSetDeployer) clientFor(obj runtime.Object) (dynamic.Resourc
 	return msd.dynamicClient.Resource(*gvr).Namespace(machineSet.GetNamespace()), nil
 }
 
-func (msd *k8sMachineSetDeployer) GetWorkerNodeImage(machineSet *unstructured.Unstructured, infraID string) (string, error) {
+func (msd *k8sMachineSetDeployer) GetWorkerNodeImage(workerNodeList []string, machineSet *unstructured.Unstructured,
+	infraID string) (string, error) {
 	machineSetClient, err := msd.clientFor(machineSet)
 	if err != nil {
 		return "", err
 	}
 
 	// TODO: After implementing a ListAll method in admiral, modify this code accordingly.
-	workerNodeList := []string{infraID + "-worker-b", infraID + "-worker-c", infraID + "-worker-d"}
```

Comment:
Can we remove the above TODO comment as it does not seem to be relevant any more?

Reply:
Moved the TODO to the respective callers.